icepyx/core/temporal.py

```diff
@@ -261,14 +269,18 @@ def validate_date_range_date(date_range, start_time=None, end_time=None):
     return _start, _end
 
 
-def validate_date_range_dict(date_range, start_time=None, end_time=None):
+def validate_date_range_dict(
+    date_range: dict[str, Union[str, dt.datetime]],
```

Comment:
```suggestion
    date_range: dict[str, Union[str, dt.date]],
```
Per the previous thread, acceptable input types are str, dt.datetime, and dt.date, so I think using dt.date captures the latter two.

Reply:
Thank you again, Jessica :bow: